app/src/main/java/org/p2p/wallet/striga/wallet/api/response/StrigaInitiateOnchainWithdrawalResponse.kt

```diff
@@ -0,0 +1,55 @@
+package org.p2p.wallet.striga.wallet.api.response
+
+import com.google.gson.annotations.SerializedName
+
+class StrigaInitiateOnchainWithdrawalResponse(
+    @SerializedName("challengeId")
+    val challengeId: String,
+    @SerializedName("dateExpires")
+    val dateExpires: String,
+    @SerializedName("transaction")
+    val transaction: Transaction,
+    @SerializedName("feeEstimate")
+    val feeEstimate: FeeEstimate,
+) {
+
+    class Transaction(
```

Comment:
I will change it to TransactionRequest/Response, because it's hard to get confused as it's a subclass. You can't just use the type TransactionRequest, you need to first specify the parent class as StrigaInitOnchainWithdrawalRequest.TransactionRequest